src/Features/Core/Portable/PullMemberUp/AbstractPullMemberUpRefactoringProvider.cs

```diff
@@ -65,7 +70,15 @@ public override async Task ComputeRefactoringsAsync(CodeRefactoringContext conte
                 return;
             }
 
-            PullMemberUpViaQuickAction(context, selectedMember, allDestinations);
+            var allActions = allDestinations.SelectAsArray(
+                destination => MembersPuller.Instance.TryComputeCodeAction(document, selectedMember, destination)).
+                WhereAsArray(action => action != null).
+                Concat(new PullMemberUpWithDialogCodeAction(document, selectedMember, this));
+
+            var nestedCodeAction = new CodeActionWithNestedActions(
+                string.Format(FeaturesResources.Pull_0_up, selectedMember.ToNameDisplayString()),
+                allActions, isInlinable: true);
```

Comment:
just to verify (since you made this inlineable), are all the titles of the nested items sensible if there is no top-level item